### ISSUE TITLE
powerline theme implicitly depends on bash's IFS

### ISSUE THREAD

**Issue by janbernloehr** [ohmybash/oh-my-bash#36]
The powerline theme implicitly assumes that bash's IFS (internal field seperator) contains a space character. This is used to iterate of the space seperated list stored in `${POWERLINE_PROMPT}` i.e.
```
  for segment in ${POWERLINE_PROMPT}; do
    local info="$(__powerline_${segment}_prompt)"
    [[ -n "${info}" ]] && __powerline_left_segment "${info}"
  done
```
In case any application changes the IFS to not contain a space, instead of a proper powerline bash prompt, one gets the error message `__powerline_user_info scm python_venv cwd_prompt: command not found`

![image](https://user-images.githubusercontent.com/1050099/48047780-7a208e80-e199-11e8-8e1c-f04a6e23b10c.png)

**Comment by Kreyren** [comment]
Assigner Important since this seems to be a blocker for powerline theme

**Comment by janbernloehr** [comment]
We have tested the fix on Ubuntu 14.04, 16.04, 18.04 as well as SLES 12.
